Src/library/base/elks/kernel/file.e

```diff
@@ -171,6 +171,39 @@ feature -- Initialization
 			open_append: is_open_append
 		end
 
+	make_temporary_open
+			-- Create a file object with a unique temporary file name,
+			-- with read/write mode.
+		do
+			make_temporary_open_with_prefix("tfn")
+		ensure
+			exists: exists
+			open_read: is_open_read
+			open_append: is_open_write
+		end
+
+	make_temporary_open_with_prefix (a_prefix: READABLE_STRING_GENERAL)
+			-- Create a file object with a unique temporary file name with prefix `a_prefix',
+			-- with read/write mode.
+		local
+			l_temp: C_STRING
+			l_res: INTEGER
+			l_ptr: POINTER
+			l_file_info: FILE_INFO
+			s: STRING
+		do
+				-- UX platforms
+				-- http://man7.org/linux/man-pages/man3/mkstemp.3.html
+			create l_temp.make (a_prefix + "XXXXXX")
+			l_res := eif_temp_file (l_temp.item)
```

Comment:
I would rename `l_res`  as `fd`  for file descriptor

Reply:
updated as `l_fd`

Reply:
I was suggesting `fd` as it is already used in similar code, and as it is short  (2 letters), it does not need the `l_` prefix.
Far from being critical.